cmd/analyzer/analyzer.go

```diff
@@ -48,10 +48,12 @@ func init() {
 	// Server options
 	fs.StringVar(&serverOpts.Host, "host", "localhost", "Hostname to bind to")
 	fs.UintVar(&serverOpts.Port, "port", 8080, "Port to listen on")
+	fs.DurationVar(&serverOpts.ShutdownGracePeriod, "shutdown-grace-perod", 15*time.Second, "Grace period to wait for server shutdown before using the force")
+	// API Handler
 	fs.StringVar(&serverOpts.APIRoot, "api-root", "/data", "Root path where to bind the API to")
+	fs.StringVar(&serverOpts.AuthURL, "auth-url", "", "Endpoint for an auth server to call for both authentication and authorization of API calls")
```

Comment:
nit: How about `auth-webhook-url` so it matches go-livepeer?

Reply:
It did go through my mnd, but then I thought it'd be more confusing since it's a different endpoint/interface altogether:
 - Auth webhook doesn't authorize any client's request headers. It is an authorized/private API, tho always called with the same infra token and it only authorizes the stream ID afaik
 - This doesn't return any info in the response, unlike all the auth webhook transcoding configs

So I feel that "auth webhook" has a different meaning/serves a different purpose today, more specific to transcoding/go-llivepeer. 

As this one could be a more generic authorization thingy (we could use from any other service as well), WDYT of naming it `auth-request-url` or `auth-server-url` instead? Then it'd be explicitly different from the `auth-webhook`.

If you feel strongly about it we can call it `auth-webhook-url` tho, lmk.

Reply:
Discussed on 1/1 and decided to keep it as `auth-url` in the end